### ISSUE TITLE
[prod] Backups for MongoDB

### ISSUE THREAD

**Issue by jakubgs** [dap-ps/infra-dapps#7]
Once #6 is done we will also need backups for MongoDB data, since even if we have 3 hosts in a cluster it can still be lost. Affects #1.

**Comment by jakubgs** [comment]
A bucket for prod backups also has been created:
https://github.com/dap-ps/infra-dapps/blob/e677f3f45fb232204d031106afb755836ec7ced3/prod.tf#L34-L38
And the scripts for backups have been deployed, but right now there is no data to backup:
```bash
admin@node-01.us-east-1a.db.prod:~ % sudo /var/lib/backups/prod_dap_ps_dump.sh      
2019-08-01T19:02:05.302+0000	dumping up to 0 collections in parallel
```
But once the Prod or Dev app is deployed and has some data I can verify this.
Other than that it's pretty much done.

**Comment by jakubgs** [comment]
I've verified that on `db.dev` host the backups work just fine:
```bash
admin@node-01.us-east-1a.db.dev:~ % sudo /var/lib/backups/dev_dap_ps_dump.sh
2019-08-05T18:10:04.922+0000	dumping up to 2 collections in parallel
2019-08-05T18:10:04.922+0000	writing dev-dap-ps.dappsmetadatas to 
2019-08-05T18:10:04.922+0000	writing dev-dap-ps.dappsimages to 
2019-08-05T18:10:04.961+0000	done dumping dev-dap-ps.dappsmetadatas (5 documents)
2019-08-05T18:10:04.962+0000	done dumping dev-dap-ps.dappsimages (4 documents)
tar: Removing leading `/' from member names

admin@node-01.us-east-1a.db.dev:~ % ls -l /var/tmp/backups/mongodb  
total 168
-rw-r--r-- 1 root root 168787 Aug  5 18:10 dev_dap_ps_dump_20190805181004.tar.gz

admin@node-01.us-east-1a.db.dev:~ % sudo -i /var/lib/backups/mongodb-backups_backup.sh
Uploading: dev_dap_ps_dump_20190805181004.tar.gz >> s3://dev-dap-ps-db-backups
upload: '/var/tmp/backups/mongodb/dev_dap_ps_dump_20190805181004.tar.gz' -> 's3://dev-dap-ps-db-backups/dev_dap_ps_dump_20190805181004.tar.gz'  [1 of 1]
 168787 of 168787   100% in    0s     6.06 MB/s  done
upload: '/var/tmp/backups/mongodb/dev_dap_ps_dump_20190805181004.tar.gz' -> 's3://dev-dap-ps-db-backups/dev_dap_ps_dump_20190805181004.tar.gz'  [1 of 1]
 168787 of 168787   100% in    0s     2.37 MB/s  done
```
Though I had to apply a few fixes:  50a14d3, 5e7cbc0, and f2024bf
